### PR TITLE
Update how-policies-and-protections-are-combined.md

### PR DESCRIPTION
**Pull request by msbemba**
Highlighted the scenario in the notes section.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5582

File: microsoft-365/security/office-365-security/how-policies-and-protections-are-combined.md [hunk 1]
```diff
@@ -57,7 +57,9 @@ There are two major factors that determine which policy is applied to a message:
 
 - **The priority of the policy**: For each type of policy (anti-spam, anti-malware, anti-phishing, etc.), there's a default policy that applies to everyone, but you can create custom policies that apply to specific users. Each custom policy has a priority value that determines the order that the policies are applied in. The default policy is always applied last.
 
-  If a user is defined in multiple policies of the same type, only the policy with the highest priority is applied to them. Any remaining policies of that type are not evaluated for the user (including the default policy).
+
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/how-policies-and-protections-are-combined.md]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**